From c8c68315f0a6b06e154b55d99cdc247cbc4a5fea Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:04:42 -0400
Subject: [usb] Use dev_is_pci() to check whether it is pci device

Message-id: <1410887463-20674-211-git-send-email-dzickus@redhat.com>
Patchwork-id: 95154
O-Subject: [RHEL7 PATCH 210/591] usb: Use dev_is_pci() to check whether it is pci device
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit e10e6f433f6f737f29dc1cea83e9a03095ac5c09
Author: Yijing Wang <wangyijing@huawei.com>
Date:   Thu Dec 5 19:21:32 2013 +0800

    usb: Use dev_is_pci() to check whether it is pci device

    Use PCI standard marco dev_is_pci() instead of directly compare
    pci_bus_type to check whether it is pci device.

    Signed-off-by: Yijing Wang <wangyijing@huawei.com>
    Acked-by: Alan Stern <stern@rowland.harvard.edu>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/host/ehci-dbg.c b/drivers/usb/host/ehci-dbg.c
index 6c8b64f..5879e9e 100644
--- a/drivers/usb/host/ehci-dbg.c
+++ b/drivers/usb/host/ehci-dbg.c
@@ -818,7 +818,7 @@ static ssize_t fill_registers_buffer(struct debug_buffer *buf)
 
 #ifdef CONFIG_PCI
  /* EHCI 0.96 and later may have "extended capabilities" */
- if (hcd->self.controller->bus == &pci_bus_type) {
+ if (dev_is_pci(hcd->self.controller)) {
   struct pci_dev *pdev;
   u32  offset, cap, cap2;
   unsigned count = 256/4;
-- 
1.7.1